### PR TITLE
Reposition tutorials in sidebar

### PR DESCRIPTION
**Pull request by murtaza-swati**
- Reposition tutorials in sidebar
- Rename "Travis CI Tutorial" button to "Travis CI OnBoarding"
- changed link for the  "Travis CI OnBoarding" button on the index page

### CODE REVIEW

**Review by vhadzh01 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <img src="images/ui/docs-desktop.svg" alt="Image of desktop">
   </figure>
   <h1>New around here? Let's get you going.</h1>
-  <p><a class="get-started-button" href="/user/tutorial/">Travis CI Tutorial</a></p>
+  <p><a class="get-started-button" href="/user/onboarding/">Travis CI OnBoarding</a></p>
```
Comment (stan-travis) [inline index.html:15]: Shouldn't it be "Onboarding" ?

**Review by stan-travis — CHANGES_REQUESTED** [review]
Onboarding instead of OnBoarding

File: index.html (hunk 1 above)
Comment (vhadzh01) [reply index.html:15]: OnBoarding / camel casing looks better

Comment (vhadzh01) [reply index.html:15]: easy to read it

Comment (stan-travis) [reply index.html:15]: Yes, please change

Comment (murtaza-swati) [reply index.html]: changed - please check now

**Review by vhadzh01 — APPROVED** [review]

**Pull request merged**